src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -11,6 +11,44 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Syntax
 {
+    internal static class Extensions
+    {
+        public static bool IsKind(this SyntaxNode? node, SyntaxKind kind1, SyntaxKind kind2)
+        {
+            return node is not null &&
+                (node.RawKind == (int)kind1 ||
+                node.RawKind == (int)kind2);
+        }
+
+        public static bool IsKind(this SyntaxNode? node, SyntaxKind kind1, SyntaxKind kind2, SyntaxKind kind3)
+        {
+            return node is not null &&
+                (node.RawKind == (int)kind1 ||
+                node.RawKind == (int)kind2 ||
+                node.RawKind == (int)kind3);
```

Comment:
@sharwell FYI this is different than the existing IsKind overloads in workspaces, which does the following:

```csharp
            var csharpKind = node.Kind();
            return csharpKind == kind1 || csharpKind == kind2 || csharpKind == kind3;
```

Note: I haven't benchmarked the differences here.